slogx.go

```diff
@@ -58,6 +58,20 @@ func Attrs(rec slog.Record) []slog.Attr {
 	return attrs
 }
 
+// HandlerAsMiddleware wrap slog.Handler as Middleware
+func HandlerAsMiddleware(handler slog.Handler) Middleware {
```

Comment:
I suggest renaming this middleware to `ApplyHandler` because the `slogm` package contains a set of middlewares, and this name better reflects its purpose. Additionally, tests for this middleware would be appreciated.

Reply:
Also, please add information about this middleware to the README.